docs/publish_yaml_s3.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+set -e
+
+CURR_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+[ -d "$CURR_DIR" ] || { echo "FATAL: no current dir";  exit 1; }
+basedir=$1
+shift
+if [[ -z ${basedir} ]] || [[ ! -d ${basedir} ]]; then
+    echo "must supply basedir as first argument"
+    exit 1
+fi
+basedir=`realpath ${basedir}`/
+
+[ -n "$AWS_ACCESS_KEY_ID"     ] || abort "AWS_ACCESS_KEY_ID is not set"
+[ -n "$AWS_SECRET_ACCESS_KEY" ] || abort "AWS_SECRET_ACCESS_KEY is not set"
+
+ver_yaml=${CURR_DIR}/yaml/versions.yml
+
+version=$(grep version ${ver_yaml} | awk ' { print $2 }')
+
+if [[ $version =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]] ; then
+    # if this is a stable version, working directory must be clean
+    # otherwise this is an rc, ea or test version and we don't care
+    if [ -n "$(git status --porcelain)" ] ; then
+        echo "working tree is dirty, aborting"
+        exit 1
+    fi
+else [[ "${BUMP_STABLE}" ]]
+    # if this isn't an X.Y.Z version, don't let allow bumping stable
+    echo "Cannot bump stable unless this is an X.Y.Z tag"
+    exit 1
+fi
+
+echo ${version} > stable.txt
+if [ -z "$AWS_BUCKET" ] ; then
+    AWS_BUCKET=datawire-static-files
+fi
+
+# make this something different than ambassador, emissary, or edge-stack
+# so we don't conflict with the new hotness we're doing for 2.0
+unversioned_base_s3_key=yaml/ambassador-docs/
```

Comment:
yeah this is a terrible path for this. other options had something to do with `legacy`... any suggestions welcome lol.